### PR TITLE
Authentik: add refreshToken method

### PR DESCRIPTION
**Pull request by RixzZ**
Added ability to refreshToken to the Authentik provider based on https://github.com/SocialiteProviders/Providers/pull/1022/commits/07f607f9c0e73b18324faa609ae6aa9bb4e0062f.

### CODE REVIEW

File: src/Authentik/Provider.php [hunk 1]
```diff
@@ -56,17 +56,32 @@ protected function getTokenUrl()
...
             ]
         );
 
         return json_decode((string) $response->getBody(), true);
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    protected function refreshToken($refreshToken)
```
Comment (atymic) [inline src/Authentik/Provider.php:73]: could we type this to string?

**Comment by atymic** [comment]
@lucasmichot I don't think linting is running correctly here?

File: src/Authentik/Provider.php (hunk 1 above)
Comment (RixzZ) [reply src/Authentik/Provider.php:73]: I guess so. I didn't type it since the rest of the provider isn't.

Should I add the type then?

**Comment by lucasmichot** [comment]
> Style fixing

Indeed it should after commit https://github.com/SocialiteProviders/Providers/pull/1081/commits/5e0ead4504b53378b085b57b6ee1ad55bbe3156f, no clue why it does not
I need a bit of time to find out

**Comment by atymic** [comment]
> > Style fixing
> 
> I does not run after every commit (`workflow_disptach`)

Can we have at least a lint on PRs?

**Comment by lucasmichot** [comment]
sure

cf. my comment https://github.com/SocialiteProviders/Manager/pull/221#issuecomment-1700790648 too

**Pull request merged**